specs/agents/tracing-instrumentation-messaging.md

```diff
@@ -176,11 +176,13 @@ incoming messages creating a transaction) and not for outgoing messaging spans.
    - Intake: key-value pairs, same like `context.request.headers`.
 - **`context.service.framework.name`**: same as `span.subtype`, but not in lowercase, e.g. `Kafka`, `RabbitMQ`
 
-#### Span destination fields
+#### Span context fields
 
 - **`context.destination.address`**: optional. Not available in some cases. Only set if the actual connection is available.
 - **`context.destination.port`**: optional. Not available in some cases. Only set if the actual connection is available.
-- **`context.destination.service.*`**: See [destination spec](tracing-spans-destination.md)
```

Comment:
The agent still needs to know to set `span.context.destination.service.{name,type}` to the empty string, right? Is it worth still linking to "tracing-spans-destination.md" for this and background info on `context.destination.service.resource`?

Reply:
Yes, that's correct.

I did some git-archeology and found that making those fields non-mandatory was done in 7.14 through [this pr](https://github.com/elastic/apm-server/pull/5468), thus I've clarified this in 37f8efe929d7f47e5fbd91ab1d38585f89033c82.